### ISSUE TITLE
Rendering html string

### ISSUE THREAD

**Issue by Fen747** [meteorhacks/meteor-ssr#41]
Hi !

`Email.send({
        to      : emails,
        from    : "noreply@talkalang.com",
        subject : obj,
        html    : SSR.render("eventMail", data)
      });`

With `data` containing a string like : "<a href='www.mysite.com'>Hello</a>".

The problem is that it is never rendered as HTML, but as a simple string, forbiding me to add links, images, <br> or any personalized element in my email body. The string comes from a text area value.

Any hint on what I am doing wrong ?

Thanks you

**Comment by sahanDissanayake** [comment]
Yes. I need to be able to inject html to the template dynamically too..

**Comment by sahanDissanayake** [comment]
I found a solution here, use `{{{ }}}` inside you template where you want html to appear
https://github.com/meteorhacks/meteor-ssr/issues/30
